lib/cli/lib/initiate.js

```diff
@@ -179,10 +179,10 @@ export default function(options, pkg) {
       default:
         paddedLog(`We couldn't detect your project type. (code: ${projectType})`);
         paddedLog(
-          "Please make sure you are running the `storybook init` command in your project's root directory."
+          "Please make sure you are running the `sb init` command in your project's root directory."
         );
         paddedLog(
-          'You can also install storybook for plain HTML snippets with `storybook init --html` or follow some of the slow start guides: https://storybook.js.org/basics/slow-start-guide/'
+          'You can also install storybook for plain HTML snippets with `sb init --html` or follow some of the slow start guides: https://storybook.js.org/basics/slow-start-guide/'
```

Comment:
These two logs were [replaced](https://github.com/storybooks/storybook/pull/4184/files#diff-3e4b1a44a581c8cf5952fefaf486682dR162) in #4184. Just a heads up depending on which PR gets merged first.

Reply:
@Keraito Merging now --- sorry to create extra work on the resolve, but my change is a small one

Reply:
No problem, won't be too much